### PR TITLE
Added frontend torch instance method and tests for `divide`

### LINKED ISSUE #20933: divide

### PR DESCRIPTION
**Pull request by iWebster28**
Close #20933

### CODE REVIEW

**Comment by iWebster28** [comment]
@aparajith21

File: ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py [hunk 1]
```diff
@@ -1403,7 +1403,15 @@ def test_torch_mul(
     fn_tree="torch.div",
     aliases=["torch.divide"],
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("numeric"),
```
Comment (aparajith21) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py:1406]: Could you change this to `available_dtypes =helpers.get_dtypes("valid")` and if there's a test failure due to dtype, add the unsupported dtype decorator appropriately?

**Comment by aparajith21** [comment]
Thanks for contributing, sorry a bit late, just checking this now

File: ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py (hunk 1 above)
Comment (iWebster28) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py:1406]: Thanks! I've gone the failing tests for test_pointwise_ops.py and test_tensor.py and added the unsupported dtypes.

Comment (aparajith21) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py:1406]: Okay great, could you please comment `add_frontend_checklist` , tick off the items there, and take a quick look to confirm that the failures have nothing to do with your code please. Will then review

**Comment by iWebster28** [comment]
add_frontend_checklist

**Comment by aparajith21** [comment]
Hi, please let me know if this is ready for review or is being worked on, and please check off the items in the list before the same!

**Comment by aparajith21** [comment]
> @aparajith21, yes, this is ready for review.

Please tick the items in the checklist as appropriate and resolve any merge conflicts, will review after that!

**Pull request closed without merging**